package.json

```diff
@@ -18,13 +13,23 @@
   "eslintConfig": {
     "extends": "react-app"
   },
+  "homepage": "https://meganandringo.com",
   "browserslist": [
     ">0.2%",
     "not dead",
     "not ie <= 11",
     "not op_mini all"
   ],
+  "dependencies": {
+    "emotion": "^10.0.9",
```

Comment:
According to Emotion's documentation, it is recommended for React project's to use the `@emotion/core` package, not the framework agnostic `emotion` package.

https://emotion.sh/docs/introduction

To use the `styled` API of the Emotion library, the `@emotion/styled` package must also be installed.

```bash
$ npm install --save @emotion/core @emotion/styled
```